salt/sdb/etcd_db.py

```diff
@@ -7,6 +7,8 @@
 :depends:       python-etcd
 :platform:      all
 
+.. versionadded:: 2015.2
+
 This module allows access to the etcd database using an ``sdb://`` URI. This
```

Comment:
I am confused. So was this added in 2015.2 or 2014.1.4 (see line 31 below). This looks like everything is referencing the same file, just in different places with different versions.